### ISSUE TITLE
Printing CLI parameters for Kubelet is misleading to users.

### ISSUE THREAD

**Issue by kannon92** [kubernetes/kubernetes#122736]
### What happened?

While working on https://github.com/kubernetes/kubernetes/pull/122241, I ran into an issue where the CLI was reporting that `--fail-swap-on="true"`. But in the code it was actually set to "false". This is because the flag was set in the KubeletConfig.

We have users also noticing that the logs in Kubelet for CLI don't report the actual value as they are actually being read from the KubeletConfig.

### What did you expect to happen?

We should bring the values of these configurations that they are rather than the CLI fields.

### How can we reproduce it (as minimally and precisely as possible)?

Log at logs for a CLI parameter (swap, eviction, etc) and change those values in the KubeletConfig. 

Kubelet logs will display the CLI parameter rather than the KubeletConfig value.

### Anything else we need to know?

https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/

CLI parameters are deprecated and replaced for KubeletConfig but I'm not sure if we bring the actual values in the logs.

### Kubernetes version

master

### Cloud provider

NA

### OS version

NA

### Install tools

<details>

</details>

### Container runtime (CRI) and version (if applicable)

Container/CRIO

### Related plugins (CNI, CSI, ...) and versions (if applicable)

<details>

</details>

**Comment by kannon92** [comment]
/sig node
/cc @tallclair 
As you mentioned in sig-node maybe help cleaning this up.

**Comment by ffromani** [comment]
/triage accepted
/priority backlog

inconsistency in the UX (cli vs config) seems very much a valid issue
I'm proposing priority backlog for starters but if we can link real user issues caused by this we can surely raise it.

**Comment by tallclair** [comment]
Yes, I can take this as part of the config cleanup.

/assign

**Comment by TessaIO** [comment]
@tallclair can I take this over?

**Comment by haircommander** [comment]
@TessaIO are you still interested in working on this?
/triage accepted

**Comment by tallclair** [comment]
Yeah, sorry, the config cleanup stalled out. Please take this over if you're interested.
/unassign
